lib/util/copyToClipboard_utils.dart

```diff
@@ -2,16 +2,7 @@
 
 import 'package:flutter/services.dart';
 
-class CopyToClipboard{
-
-
-  static Future<void> copyText(String text) async {
-    if (text.isNotEmpty) {
-      Clipboard.setData(ClipboardData(text: text));
-      return;
-    } else {
-      throw ('Please enter a string');
-    }
-  }
-
+Future<void> copyText(String text) async {
```

Comment:
my point was why even have this class at all? It's a whole class that makes exactly 1 call. Remove this class and use Clipboard.setData directly.

Reply:
its done